src/main/java/core/basesyntax/classes/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.classes.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int shortBase;
+    private final int longBase;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int shortBase, int longBase, int height) {
+        super(color);
+        if (shortBase < longBase) {
+            this.shortBase = shortBase;
+            this.longBase = longBase;
+        } else {
+            this.shortBase = longBase;
+            this.longBase = shortBase;
+        }
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) this.height * (this.shortBase + this.longBase) / 2;
+    }
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.